activestorage/lib/active_storage/service/azure_storage_service.rb

```diff
@@ -10,8 +10,8 @@ module ActiveStorage
   class Service::AzureStorageService < Service
     attr_reader :client, :blobs, :container, :signer
 
-    def initialize(storage_account_name:, storage_access_key:, container:)
-      @client = Azure::Storage::Client.create(storage_account_name: storage_account_name, storage_access_key: storage_access_key)
+    def initialize(storage_account_name:, storage_access_key:, container:, **extra_client_options)
```

Comment:
Let’s call the splat argument `options`, like we do in the S3 service.

Reply:
Hi George,
  I have renamed that to options now

Thanks for the feedback